README.md

```diff
@@ -259,17 +258,39 @@ will be converted to `width="100"`.
 The class can be used like this:
 
 ```php
-$converter = new \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter($rawHtml);
+$converter = \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter::fromHtml($rawHtml);
+$visualHtml = $converter->convertCssToVisualAttributes()->render();
+```
+
+You can also have the `CssToAttributeConverter` work on a `DOMDocument`:
+
+```php
+$converter = \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter::fromDomDocument($domDocument);
 $visualHtml = $converter->convertCssToVisualAttributes()->render();
 ```
 
 ### Technology preview of new classes
```

Comment:
Again, this was in 2.1, but I wonder if it should be clearer that this applies to all the classes in this section, "Processing HTML".  I guess we'll find out after 2.2...

Reply:
I've rearranged the sections a bit now.